js/src/lib/interfaces/Libraries.ts

```diff
@@ -4,25 +4,25 @@ import type { ModelData } from "./Types";
  * Add your new library here.
  */
 export enum ModelLibrary {
-	"adapter-transformers"   = "Adapter Transformers",
```

Comment:
There are quite a few unrelated changes in the PR (maybe your code editor), do you think you could revert those?

Reply:
Should be OK now.